packages/bundle-size-checker/src/uploadSnapshot.js

```diff
@@ -12,6 +12,22 @@ async function getCurrentCommitSHA() {
   return stdout.trim();
 }
 
+/**
+ * Sanitizes a string to be used as an S3 tag value
+ * See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html#tag-restrictions
+ * @param {string} str
+ * @returns {string}
+ */
+function sanitizeS3TagString(str) {
+  // Replace disallowed characters with underscore
+  let safe = str.replace(/[^a-zA-Z0-9 +-=.:/@]/g, '_');
+  safe = safe.replace(/_+/g, '_'); // Replace multiple underscores with single
```

Comment:
Wouldn't this be enough?

```suggestion
  const safe = str.replace(/[^a-zA-Z0-9 +-=.:/@]+/g, '_');
```

Reply:
Will need to add `_` in the regex then, to catch things like `'foo_+_bar'`